### PR TITLE
updated Azure platform setup page

### PR DESCRIPTION
**Pull request by phillipgibson**
Please provide a description for what this PR is for.

Updated the Azure Platform Setup page to include tip for AKS Istio managed add-on. 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by phillipgibson** [comment]
I believe I've signed all the CLA info.

**Comment by kfaseela** [comment]
PR Opener: @phillipgibson
Author: <phill**********ithub​@outlook.com>. could you please confirm if the CLA signed is linked with this mail id?

**Comment by phillipgibson** [comment]
> PR Opener: @phillipgibson Author: <phill**********ithub​@outlook.com>. could you please confirm if the CLA signed is linked with this mail id?

I've updated my email to match the email in the CLA. Not exactly sure how to purge the original for the test to pass.

File: content/en/docs/setup/platform-setup/azure/index.md [hunk 1]
```diff
@@ -4,19 +4,24 @@ description: Instructions to set up an Azure cluster for Istio.
...
+  - /docs/setup/kubernetes/platform-setup/azure/
+keywords: [platform-setup, azure]
 owner: istio/wg-environments-maintainers
 test: no
 ---
 
-This page was last updated September 12, 2019.
-
-{{< boilerplate untested-document >}}
+This page was last updated April 26, 2023
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/azure/index.md:14]: I'm not sure you need to put the date in here, as we don't have it on the other pages. I believe there was no known testing of this page at one point in time, so the two lines were added so users would know the page may be stale. I think the plan at that time was to simply have the page updated and remove the two lines.

Comment (phillipgibson) [reply content/en/docs/setup/platform-setup/azure/index.md:14]: Ok I'll remove that. It was there previously, and I didn't know the context of it.

**Pull request closed without merging**